### PR TITLE
Story#644

### PR DESCRIPTION
**Pull request by rejneesh1**

### CODE REVIEW

File: config.xml [hunk 1]
```diff
@@ -1,9 +1,9 @@
 <?xml version='1.0' encoding='utf-8'?>
-<widget id="org.shikshalokam.samiksha.darpan" version="2.1.9" xmlns="http://www.w3.org/ns/widgets" xmlns:android="http://schemas.android.com/apk/res/android" xmlns:cdv="http://cordova.apache.org/ns/1.0">
-    <name>Darpan</name>
+<widget id="org.shikshalokam.samiksha" version="2.1.9" xmlns="http://www.w3.org/ns/widgets" xmlns:android="http://schemas.android.com/apk/res/android" xmlns:cdv="http://cordova.apache.org/ns/1.0">
+    <name>Samiksha</name>
```
Comment (kiranharidas187) [inline config.xml]: there should not be change in this file

File: config.xml [hunk 2]
```diff
@@ -208,7 +192,35 @@
     <allow-navigation href="http://localhost:8100" sessionid="36b632ef" />
     <plugin name="cordova-plugin-fcm-with-dependecy-updated" spec="^4.1.0" />
     <plugin name="cordova-plugin-local-notification" spec="^0.9.0-beta.2" />
+    <allow-navigation href="http://192.168.43.184:8100" />
```
Comment (kiranharidas187) [inline config.xml]: revert all changes from this file

File: src/pages/forgot-password/forgot-password.html [hunk 3]
```diff
@@ -1,11 +1,11 @@
 <ion-content padding class="mainConatiner">
   <div class="container _flex-box">
     <div class="_flex-box _justify-content-center imageConatiner">
-      <img src="assets/imgs/image1.jpg" class="imageAdj">
+      <img src="assets/imgs/image1.jpg" class="imageAdj" />
```
Comment (kiranharidas187) [inline src/pages/forgot-password/forgot-password.html:4]: unwanted change

File: src/pages/forgot-password/forgot-password.ts [hunk 4]
```diff
@@ -1,108 +1,49 @@
...
 
-  getPassword() {
-    const params = new HttpParams({
-      fromObject: {
-        staffID: this.payload['staffID'],
-        registeredMobileNo: this.payload['registeredMobileNo'],
-        key: AppConfigs.punjabApiKey
+  forgetPassword() {
+    let payload = this.forgotPasswordForm.value;
+    let url = AppConfigs.user_management_base_url + "v1" + AppConfigs.punjab.forgotPasswordNew;
```
Comment (kiranharidas187) [inline src/pages/forgot-password/forgot-password.ts]: git v1 in AppConfigs.punjab.forgotPasswordNew

File: src/pages/reset-password/reset-password.ts [hunk 5]
```diff
@@ -1,138 +1,63 @@
...
-        } else if (xmlDoc.getElementsByTagName('string')[0].childNodes[0].nodeValue.indexOf('Your password has chanced !!!') >= 0) {
-          this.utils.openToast("Your password has been changed");
-          this.navCtrl.pop();
-        } else {
-          this.utils.openToast("Invalid password.");
-        }
+  resetPasswordNew() {
+    let payload = this.resetPasswordForm.value;
+    payload.facultyCode = this.partnerDetails.facultyCode;
+    let url = AppConfigs.user_management_base_url + "v1" + AppConfigs.punjab.resetPasswordNew;
```
Comment (kiranharidas187) [inline src/pages/reset-password/reset-password.ts]: put v1 in AppConfigs.punjab.resetPasswordNew

File: src/pages/user-login/user-login.ts [hunk 6]
```diff
@@ -1,275 +1,101 @@
...
-        this.utils.openToast("Unable to login. Please try again");
-        this.isNewUser = true;
-      });
-  }
+  // only using this to login now .
+  loginNew() {
+    this.utils.startLoader();
 
-  adminLogin() {
     const params = new HttpParams({
```
Comment (kiranharidas187) [inline src/pages/user-login/user-login.ts]: Do you need this code??

File: src/pages/forgot-password/forgot-password.html (hunk 3 above)
Comment (rejneesh1) [reply src/pages/forgot-password/forgot-password.html:4]: img is self closing tag so its coming here by prettier.

**Pull request merged**